recipes/igblast/build.sh

```diff
@@ -1,45 +1,74 @@
 #!/bin/bash
-set -euo pipefail
+set -xeuo pipefail
+
+# This script uses ideas from the build script for BLAST. See comments there.
 
-IGBLAST_ADDRESS=ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release
 SHARE_DIR=$PREFIX/share/igblast
 
 mkdir -p $PREFIX/bin
 
-if [ $(uname) == Linux ]; then
-  # The binaries want libbz2.so.1, but the correct soname is libbz2.so.1.0
-  for name in makeblastdb igblastn igblastp; do
-    patchelf --replace-needed libbz2.so.1 libbz2.so.1.0 bin/$name
-  done
-fi
-
 # $SHARE_DIR contains the actual igblastn and igblastp binaries and also the
 # required data files. Wrappers will be installed into $PREFIX/bin that set
 # $IGDATA to point to those data files.
 mkdir -p $SHARE_DIR/bin
 
-# Copy binaries and wrappers
-for name in igblastn igblastp; do
-  mv bin/$name $SHARE_DIR/bin/
-  sed "s/igblastn/$name/g" $RECIPE_DIR/igblastn.sh > $PREFIX/bin/$name
-  chmod +x $PREFIX/bin/$name
-done
+if [[ $(uname) == Linux ]]; then
+    export CFLAGS="$CFLAGS -O2"
+    export CXXFLAGS="$CXXFLAGS -O2"
+    export CPPFLAGS="$CPPFLAGS -I$PREFIX/include -I$PREFIX/include/ncbi-vdb"
+    export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
```

Comment:
We actually don't need to add the `-O2`, `-I$PREFIX/include` and `-L$PREFIX/lib` flags since they are already set by the compiler activation scripts.
We can remove those when the recipe gets updated again -- no need to run the hour long CI job just for that right now!